napari/utils/_tests/test_io.py

```diff
@@ -82,9 +82,9 @@ def raise_no_bigtiff(*args, **kwargs):
 
     monkeypatch.setattr(tifffile, 'imwrite', raise_no_bigtiff)
 
-    data = np.random.randint(
-        low=0, high=2**16, size=(128, 4096, 4096), dtype='uint16'
-    )  # 4GB size
+    # create image data, it could be smaller, as we patch the tifffile.imwrite
+    # function to raise an error if bigtiff is not set
```

Comment:
```suggestion
    # create image data. It can be <4GB compressed because we
    # monkeypatch tifffile.imwrite to raise an error if bigtiff is not set
```